rambo/vagrant/modules.rb

```diff
@@ -29,6 +29,10 @@ def read_provider_file
 def write_provider_file(provider)
   tmp_dir = get_env_var_rb('TMPDIR_PATH') || File.join(Dir.pwd,  '.' + PROJECT_NAME + '-tmp')
   provider_path = File.join(tmp_dir, 'provider')
+  dirname = File.dirname(provider_path)
+  unless File.directory?(dirname)
+    FileUtils.mkdir_p(dirname)
+  end
```

Comment:
Bonus bugfix?

It looks like if you specify a tmpd_dir that doesn't exist, it gets created here, so it can be used, right? What happens if you don't? The use case is when you set tmp_dir manually to a path that doesn't exist (yet), right?

This can be shortened a bit by removing `dirname = File.dirname(provider_path)` and just operating on `tmp_dir`, since it's the same thing.